CMakeLists.txt

```diff
@@ -24,6 +24,8 @@
 ################################################################################
 
 cmake_minimum_required ( VERSION 3.5.0 )
+set(CMAKE_SHARED_LINKER_FLAGS_INIT "-Wl,--enable-new-dtags,--rpath,$ORIGIN:$ORIGIN/.." CACHE STRING "RUNPATH for libraries")
```

Comment:
Any reason these variables are set before the project() ?

Reply:
The  linker flags can be set before and after the project  **without using as a cache** 
But for using the same as cache variable , it need to be set before the project
https://stackoverflow.com/questions/32698580/cmake-global-linker-flag-setting-for-all-targets-in-directory

Reply:
Ok. Also please add a comment on why these variables are required.

Reply:
Upon build and rvs package installation, rvs bin is not find libyaml-cpp.so.0.7 path even though the lib is present.
	**libyaml-cpp.so.0.7 => not found** - Observed
	**libyaml-cpp.so.0.7 => /usr/local/lib64/libyaml-cpp.so.0.7 (0x00007f2d0b42c000)** - Expected

Before **CMAKE_INSTALL_RPATH_USE_LINK_PATH** was helping in setting this search path in rpath. In this commit, it was removed.